bescort.lic

```diff
@@ -199,14 +199,32 @@ class Bescort
       ],
       [
         { name: 'hvaral_passport', regex: /hvaral_passport/i, description: 'Handles the gate between Hvaral and Muspari' }
+      ],
+      [
+        { name: 'brocket_1', regex: /brocket_1/i, description: 'Enter the young brocket deer hunting area.' },
```

Comment:
❓ What do you think about `young_brocket`, etc. here instead of 1/2/3?

Reply:
I was trying to match the rest of the hunting areas and how we do them.

Reply:
lol, we actually have a mix of it. I changed it to young, mid, and elder.